services/slam/builtin/builtin_test.go

```diff
@@ -56,6 +56,8 @@ var (
 	orbslamIntSynchronizeCamerasChan          chan int = make(chan int)
 	cartographerIntLidarReleasePointCloudChan chan int = make(chan int, 1)
 	validMapRate                                       = 200
+	trueVar                                            = true
```

Comment:
Accepting better recommendations for variable names! 

Note: I thought we could potentially use these for any fields where an address to a pointer is needed and since `use_live_data` is required I didnt want to have to define it in every test

Reply:
My only idea is maybe `true_` and `false_`? But I don't have a strong preference.

Reply:
useDataFlag?

Reply:
o misread, prob `useLiveTrue` and `useLiveFalse`

Reply:
After offline chats, I'm chosen to go with _true and _false, feel free to add comments if you disagree